### PR TITLE
Om56

### PR DESCRIPTION
**Pull request by mphanias**
modified multi-cluster-view dashboard to display server topology using cluster-name and xdr-dc configs
updated docker-compose and AeroLab create scripts to include required plugin

### CODE REVIEW

**Review by hev — APPROVED** [review]
I hit an error trying this out, which I realize now was that I didn't have the plugin. Maybe we can call this out better somehow. 

Multi_Cluster_View_-_Aerospike_-_Dashboards_-_Grafana_🔊

With Aerolab (locally w/docker) it looks nice OOB. I approve but we should look into automating the plugin install if possible.

**Comment by mphanias** [comment]
The plugin is diagram-plugin, AeroLab/create.sh script was to pre-install the plugin ( marked below) 
is there any other place we need to modify ?

aerolab client create ams -n ams -s dc1,dc2 --instance e2-medium --zone us-central1-a --disk pd-balanced:20 -e 3000:3000
aerolab client attach  -n ams -- grafana-cli plugins install grafana-polystat-panel
**aerolab client attach  -n ams -- grafana-cli plugins install jdbranham-diagram-panel**
aerolab client attach  -n ams -- service grafana-server restart

File: config/grafana/dashboards/geoview.json [hunk 1]
```diff
@@ -62,293 +68,284 @@
...
+      "title": "Data Replication topology",
       "type": "row"
     },
     {
       "datasource": {
         "type": "prometheus",
         "uid": "${DS_AEROSPIKE_PROMETHEUS}"
       },
-      "description": "Displays the number of aerospike nodes configured within a cluster, in case if there is a condition like split-brain or different cluster-sizes reported, this panel displays different values indicating issues",
+      "description": "Displays topology of the connected aerospike servers across data centers, this diagram panel is dependent on cluster-name and the xdr names configured in the aerospike configuration files.",
```
Comment (hev) [inline config/grafana/dashboards/geoview.json:342]:
```suggestion
      "description": "Displays topology of the connected aerospike servers across data centers, this diagram panel is dependent on cluster-name and the xdr names configured in the aerospike configuration files. This panel requires the diagram-panel plugin.",
```

**Comment by mphanias** [comment]
updated the description as suggested to include info about the diagram plugin

**Pull request merged**